app/views/payments/openPayments.scala.html

```diff
@@ -25,9 +25,7 @@
 @hiddenHelpTextContent = {
   <p>@Messages("openPayments.timeToAppear")</p>
   <p>@Messages("payment.stillMakeLink")
-      <a href="@appConfig.unauthenticatedPaymentsUrl" id="vatPaymentsLink">
-     @messages("payment.stillMake")
-      <span class="visuallyhidden">@messages("payment.evenIfNotShown")</span></a>.
+    <a href="@appConfig.unauthenticatedPaymentsUrl" id="vatPaymentsLink">@messages("payment.stillMake")<span class="visuallyhidden">@messages("payment.evenIfNotShown")</span></a>.
```

Comment:
Can you add a space between `@messages("payment.stillMake")` and `<span class=visuallyhidden...`